test/unit/modules/record-events-browser-spec.js

```diff
@@ -47,14 +48,37 @@ describe('.recordEvent(s) methods (browser)', () => {
   });
 
   describe('.recordEvent', () => {
-
-    it('should not send events if set to \'false\'', () => {
+    it('should not send events if KeenTracking.enabled is set to \'false\'', () => {
       KeenTracking.enabled = false;
       client.recordEvent('not-going', { test: 'data' }, mockFn1);
       expect(mockFn1).toBeCalledWith(expect.any(String), null);
       KeenTracking.enabled = true;
     });
 
+    it('should not send events if KeenTracking.optedOut is set to \'true\'', () => {
+      KeenTracking.optedOut = true;
+      client.recordEvent('not-going', { test: 'data' }, mockFn1);
+      expect(mockFn1).toBeCalledWith(expect.any(String), null);
```

Comment:
let's change that line to :
```
expect(mockFn1).not.toBeCalled();
```
because we won't send any info about error nor response